.env.example

```diff
@@ -22,3 +22,12 @@ STORAGE_SECRET=
 # Unique identifier for this instance, used in the Atlassian Connect manifest to
 # differentiate this instance from other deployments (staging, dev instances, etc).
 # INSTANCE_NAME=<you-username>
+
+# Unique identifier for whether this env will be used in Github.com or Github AE. (github.com, ghae etc)
+GITHUB_INSTANCE=
```

Comment:
add comment like this this flag is getting used to define the code path as per instance type like if( GHAE ) { } if ( .com )